client/src/components/HeaderComponent/style.css

```diff
@@ -0,0 +1,35 @@
+Button > div {
+  width: 4rem;
+  height:0.35rem;
+  background-color: white;
+  margin: 1rem 0;
+}
+.burger{
```

Comment:
why burger? 😃 pls rename to something meaningful (in BEM this could be `header__button`, with the top level div having class `header`)

Reply:
ok I assume it's gonna be the menu icon, think simply `header__menu-icon` or `header__menu-button` (since it's a button) would be more clear..